bin/upload_sarif_to_github

```diff
@@ -3,8 +3,8 @@
 set -euo pipefail
 
 sarif_file="${1:-}"
-owner=$(basename "$(dirname "${BUILDKITE_PULL_REQUEST_REPO}")")
-repo=$(basename "${BUILDKITE_PULL_REQUEST_REPO%.git}")
+owner=$(echo "$BUILDKITE_REPO" | awk -F'[:/]' '{print $2}')
```

Comment:
This splits into 
```
git@github.com
Automattic
Gravatar-SDK-Android.git
```

so the second one will be the owner

Reply:
I mean, assuming: `$BUILDKITE_REPO=git@github.com:Automattic/Gravatar-SDK-Android.git`